lib/Rex/Box/Base.pm

```diff
@@ -196,7 +196,7 @@ Sets the memory of a VM in megabyte.
 
 sub memory {
   my ( $self, $mem ) = @_;
-  $self->{memory} = $mem;
+  $self->{memory} = $mem * 1024; # libvirt wants kilobytes
```

Comment:
What happens if Box is asked to work with VirtualBox backend instead of LibVirt? Is it correct to always multiply the passed memory argument by 1024?